### PR TITLE
Add Facebook Login

### PR DESCRIPTION
**Pull request by estacioneto**
facebook login

### CODE REVIEW

File: client/sagas/user.js [hunk 1]
```diff
@@ -43,10 +47,44 @@ export function* userLoginSaga() {
 
   const { accessToken, idToken } = login
 
+  yield call(firebaseLoginAndRedirect, 'Google', idToken, accessToken)
```
Comment (lucasecdb) [inline client/sagas/user.js]: I think you don't need to use `call` here, you could just do
```suggestion
  yield* firebaseLoginAndRedirect('Google', idToken, accessToken)
```
The same applies to the facebook saga

File: client/actions/user.js [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
   type: USER_REQUEST_LOGOUT,
 })
 
-export const userLogin = (user) => ({
+export const userRequestFacebookLogin = token => ({
+  type: USER_REQUEST_LOGIN_FACEBOOK_WITH_TOKEN,
+  token,
+})
+
+export const userLogin = user => ({
```
Comment (lucasecdb) [inline client/actions/user.js]: Can you rename this action to `userLoggedIn`? It's kinda confusing whether this is a login request or if a user logged in successfully.

**Comment by lucasecdb** [comment]
Other thing, I think we need some way to figure out if the user already has an account with google and link the google and facebook profiles with it, instead of creating a new user.

**Comment by estacioneto** [comment]
Maybe, @lucasecdb ... Maybe in the future...

**Pull request merged**